third-party/esbuild-plugins-polyfills/README.md

```diff
@@ -0,0 +1,7 @@
+# @esbuild-plugins/node-modules-polyfill
+
+This is `@esbuild-plugins/node-modules-polyfill`, with the following changes:
+
+- using a better library for polyfills as proposed by this PR which has not merged: https://github.com/remorses/esbuild-plugins/pull/19
+- There are a couple correctness changes on top of the above PR, as described here: https://github.com/remorses/esbuild-plugins/pull/19#discussion_r990493080
```

Comment:
Seems like both this issues are resolved now? Can we use the original library?

Reply:
Can you update this description then, since the PR is merged but the issue is still unresolved